operations/research-and-development/product/development-process/user-interface-text-guidelines.md

```diff
@@ -84,47 +93,113 @@ An error message informs the user that something went wrong. Errors prevent the
 * If a solution or workaround isn’t obvious, suggest one.
 * Examples:
   * **Messages must have fewer than 120 characters.**
-  * **A connection to the Plugin Marketplace server could not be established. Check your settings in the** [**System Console**](https://github.com/mattermost/mattermost-handbook/tree/7aa7b57d703dc9e728d9ba53931bae8f5585d8f3/operations/research-and-development/product/hyperlink%3E/README.md). 
+  * **A connection to the Marketplace server could not be established. Check your settings in the **System Console**.**
 
-## Button labels and links
+## In-product copy
 
-Button labels should always use action words and describe the action as concisely as possible. They should be limited to four words or less. Examples: “Log In”, “Send Invitation”.
+The Technical Writers often work with the UI/UX team on in-product copy. The majority of guidelines are available in the [Mattermost Compass Design System](https://zeroheight.com/29be2c109/p/995619-compass-design-system).
 
-## Navigation labels
+### Guidelines for punctuation
```

Comment:
I'm not entirely sure we need this section - might be better to replace it with a sentence instead?

Reply:
I wonder if a table would work well here? 4 columns: guideline, description, good example, bad example

Reply:
Oh yes, that's a great idea! Will add that into the next iteration, thanks!